spec/spec.md

```diff
@@ -1390,28 +1390,60 @@ format-related rules above:
     },
     "field": {
       "type": "object",
-      "properties": {
-        "path": {
-          "type": "array",
-          "items": { "type": "string" }
+      "oneOf": [
+        {
+          "properties": {
+            "path": {
+              "type": "array",
+              "items": { "type": "string" }
+            },
+            "purpose": { "type": "string" },
+            "filter": {
+              "type": "object",
+              "properties": {
+                "type": { "type": "string" },
+                "format": { "type": "string" },
+                "pattern": { "type": "string" },
+                "minimum": { "type": "string" },
+                "minLength": { "type": "integer" },
+                "maxLength": { "type": "integer" }
+              },
+              "required": ["type"],
+              "additionalProperties": false
+            }
+          },
+          "required": ["path"],
+          "additionalProperties": false
         },
-        "purpose": { "type": "string" },
-        "filter": {
-          "type": "object",
+        {
           "properties": {
-            "type": { "type": "string" },
-            "format": { "type": "string" },
-            "pattern": { "type": "string" },
-            "minimum": { "type": "string" },
-            "minLength": { "type": "integer" },
-            "maxLength": { "type": "integer" }
+            "path": {
+              "type": "array",
+              "items": { "type": "string" }
+            },
+            "purpose": { "type": "string" },
+            "filter": {
+              "type": "object",
+              "properties": {
+                "type": { "type": "string" },
+                "format": { "type": "string" },
+                "pattern": { "type": "string" },
+                "minimum": { "type": "string" },
+                "minLength": { "type": "integer" },
+                "maxLength": { "type": "integer" }
```

Comment:
should this list of properties also include: `exclusiveMinimum`, `exclusiveMaximum`, `maximum`, `const`, `not`, and `enum`?

Reply:
yes you're correct. thanks for pointing that out.

Reply:
Also updated the suggestion here to include strings on the min/max comparisons.